src/matrices.jl

```diff
@@ -169,7 +171,18 @@ Quantification Analysis. Theory and Best Practices*, Springer, pp. 3-43 (2015).
 """
 function RecurrenceMatrix(x, ε; metric = DEFAULT_METRIC, kwargs...)
     m = getmetric(metric)
+<<<<<<< HEAD
```

Comment:
this has to be a mistake, along with the other `=====` stuff from resolving conflicts.

Reply:
fixed